snowplow/iglu-client-embedded/schemas/com.metabase/task/jsonschema/1-0-0

```diff
@@ -61,7 +61,7 @@
       "minimum": 0,
       "maximum": 2147483647
     },
-    "task-details": {
+    "task_details": {
```

Comment:
I contacted Joan to help me make this simple change on Snowcatplow without adding a new schema.